api/envoy/config/cluster/aggregate/cluster.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package envoy.config.cluster.aggregate;
+
+option java_outer_classname = "ClusterProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.cluster.aggregate";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Aggregate cluster configuration]
+
+// Configuration for the aggregate cluster. See the :ref:`architecture overview
+// <arch_overview_aggregate_cluster>` for more information.
+message ClusterConfig {
+  // Load balancing clusters in aggregate cluster with explicit priority.
+  message LbCluster {
+    // Cluster name.
+    string cluster = 1;
+
+    // Cluster priority.
+    int32 priority = 2;
```

Comment:
Please make this uint32, we don't use signed types unless needed in Envoy API style.

Reply:
Done